migrations/Version20220920132618.php

```diff
@@ -10,7 +10,7 @@
 /**
  * Auto-generated Migration: Please modify to your needs!
  */
-final class Version20220915083555 extends AbstractMigration
+final class Version20220920132618 extends AbstractMigration
```

Comment:
You should be careful with renaming migration classes. The timestamp, e.g. “20220920132618”, is used to keep track of which migrations have been run. 

However, you have to regenerate migrations when switching to MariaDB. This is not at big issue right now for this project, but if we already had the project running out in the wild considerations on how to handle this should be made.